slices/libxml-utils.yaml

```diff
@@ -0,0 +1,17 @@
+package: libxml2-utils
```

Comment:
The filename for this should be `libxml2-utils.yaml` instead of `libxml-utils.yaml`.